.github/workflows/version-bump.yml

```diff
@@ -22,33 +19,19 @@ on:
       version_number:
        description: 'The version number to bump to'
```

Comment:
Can you add an example format here in parens?  Makes it easier to know if a dash should be included or not.